Projects/Foundations/Schemas/Docs-Tutorials/D/Developers/Developers-Intro/developers-intro-020-tutorial-topic-files-component.json

```diff
@@ -77,11 +103,15 @@
             "text": "Daily Files are managed by a File Cursor which dynamically loads the files of one day and its neighboring days.",
             "translations": [
                 {
-                    "language": "RU",
+                    "language": "DE",
+                    "text": "Tägliche Dateien werden von einem Dateicursor verwaltet, der dynamisch die Dateien eines Tages und seiner Nachfolgetage lädt.",
+                    "updated": 1641736822077
+        },
+        {                    "language": "RU",
                     "text": "Daily Files управляются File Cursor, который динамически загружает файлы одного дня и соседних с ним дней.",
                     "updated": 1640932087275
                 }
```

Comment:
One of the parts I would afterwards change.

Reply:
have lined up the brackets